config/settings.yml

```diff
@@ -1081,7 +1081,8 @@ va_mobile:
   url: "https://veteran.apps.va.gov"
   ppms_base_url: "https://staff.apps.va.gov"
   mock: false
-  key_path: /fake/client/key/path
+  # key_path: /fake/client/key/path
```

Comment:
Did you mean to leave this in?

Reply:
Removed this unintentionally commited file and also cleaned out other dead code